Src/RE2.Managed/Regex2.cs

```diff
@@ -219,7 +220,11 @@ public static void ClearRegexes()
         ///
         /// </code>
         /// </example>
-        public static unsafe bool Matches(string pattern, string text, out List<Dictionary<string, string>> matches)
+        public static unsafe bool Matches(
+            string pattern,
+            string text,
+            out List<Dictionary<string, string>> matches,
+            out List<Dictionary<string, (int Start, int Length)>> matchIndices)
```

Comment:
>int Start, int Length [](http://example.com/codeflow?start=41&length=21)

instead of doing this, pls, use the FlexMatch (check my other pr)

Reply:
Done.